[2022-10-10T15:55:50.909+0600] {processor.py:156} INFO - Started process (PID=441) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:55:50.910+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:55:50.910+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:55:50.910+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:55:50.929+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:55:50.927+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:55:50.930+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:55:50.950+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.044 seconds
[2022-10-10T15:56:22.230+0600] {processor.py:156} INFO - Started process (PID=465) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:56:22.231+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:56:22.232+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:56:22.231+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:56:22.257+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:56:22.255+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:56:22.257+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:56:22.280+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.055 seconds
[2022-10-10T15:56:53.409+0600] {processor.py:156} INFO - Started process (PID=489) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:56:53.412+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:56:53.413+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:56:53.413+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:56:53.431+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:56:53.429+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:56:53.431+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:56:53.454+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.055 seconds
[2022-10-10T15:57:24.048+0600] {processor.py:156} INFO - Started process (PID=513) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:57:24.048+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:57:24.049+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:57:24.049+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:57:24.067+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:57:24.065+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:57:24.067+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:57:24.095+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.051 seconds
[2022-10-10T15:57:54.887+0600] {processor.py:156} INFO - Started process (PID=537) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:57:54.888+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:57:54.888+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:57:54.888+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:57:54.906+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:57:54.904+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:57:54.906+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:57:54.929+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.045 seconds
[2022-10-10T15:58:25.706+0600] {processor.py:156} INFO - Started process (PID=561) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:58:25.707+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:58:25.708+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:58:25.708+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:58:25.724+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:58:25.723+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:58:25.725+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:58:25.746+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.044 seconds
[2022-10-10T15:58:56.653+0600] {processor.py:156} INFO - Started process (PID=585) to work on /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:58:56.655+0600] {processor.py:758} INFO - Processing file /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py for tasks to queue
[2022-10-10T15:58:56.656+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:58:56.655+0600] {dagbag.py:525} INFO - Filling up the DagBag from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:58:56.680+0600] {logging_mixin.py:117} INFO - [2022-10-10T15:58:56.677+0600] {dagbag.py:330} ERROR - Failed to import: /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
Traceback (most recent call last):
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py", line 9, in <module>
    from airflow.operators.mysql_operator import MySqlOperator
  File "/home/saima/.local/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.mysql'
[2022-10-10T15:58:56.680+0600] {processor.py:770} WARNING - No viable dags retrieved from /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py
[2022-10-10T15:58:56.704+0600] {processor.py:178} INFO - Processing /d/Saima/Data-Engineering/Apache-Airflow/DAG/dags/aqm_hrms_total_data_extractor.py took 0.059 seconds
